sunpy/map/sources/suit.py

```diff
@@ -0,0 +1,91 @@
+"""
+SUIT Map sub class definitions
+"""
+from astropy.visualization import AsinhStretch, ImageNormalize
+
+from sunpy.map.mapbase import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+
+__all__ = ["SUITMap"]
+__author__ = ["Rahul Gopalakrishnan"]
+__email__ = ["rahulg.astro@gmail.com"]
```

Comment:
```suggestion
```

We do not typically add author and email information in the file directly anymore.